test/e2e/container_probe.go

```diff
@@ -41,9 +41,22 @@ var _ = Describe("Probing container", func() {
 	It("with readiness probe should not be ready before initial delay and never restart [Conformance]", func() {
 		p, err := podClient.Create(makePodSpec(probe.withInitialDelay().build(), nil))
 		expectNoError(err)
-		startTime := time.Now()
 
-		Expect(wait.Poll(poll, 90*time.Second, func() (bool, error) {
+		Expect(wait.Poll(poll, 120*time.Second, func() (bool, error) {
+			p, err := podClient.Get(p.Name)
+			if err != nil {
+				return false, err
+			}
+
+			if p.Status.Phase != api.PodRunning {
+				Logf("pod is not yet running; pod has phase %q.", p.Status.Phase)
+				return false, nil
+			}
+			return true, nil
+		})).NotTo(HaveOccurred(), "pod never became running")
+
+		startTime := time.Now()
```

Comment:
My concern is that this will just start more bleeding somewhere else. I suspect the flake caused by this will be more frequent than the flake this is fixing.